### PR TITLE
Fixed sheet_name_index in transifex

### PR DESCRIPTION
**Pull request by orangejenny**
PR into https://github.com/dimagi/commcare-hq/pull/23658

https://github.com/dimagi/commcare-hq/pull/23658#pullrequestreview-222932861

(product team, ignore this)

### CODE REVIEW

File: corehq/apps/translations/generators.py [hunk 1]
```diff
@@ -254,6 +254,7 @@ def _get_translation_for_sheet(self, app, sheet_name, rows):
             unique_id_index = self._get_header_index(MODULES_AND_FORMS_SHEET_NAME, 'unique_id')
 
             def occurrence(_row):
+                sheet_name_index = sheet_name_index.replace("menu", "module")
```
Comment (mkangia) [inline corehq/apps/translations/generators.py]: the correct change here would be
`return ':'.join([_row[type_index], _row[sheet_name_index].replace("menu", "module"), _row[unique_id_index]])`

**Review by mkangia — COMMENTED** [review]
please dont merge this yet. I speculated that this would be needed but i am yet to confirm.

File: corehq/apps/translations/generators.py (hunk 1 above)
Comment (mkangia) [reply corehq/apps/translations/generators.py]: We would need to keep the type as "Module" so this should be,
`return ':'.join([_row[type_index].replace("menu", "Module"), _row[sheet_name_index].replace("menu", "module"), _row[unique_id_index]])`

**Comment by mkangia** [comment]
Tested the integration and updated.
one thing we would need to do when we roll out this change is to rename file on transifex from "Modules_and_forms" to "Menu_and_forms". Thats the only file that is mapped by its name. Its a quick change and i can take action on that. Please just let me know when this is deployed to India.
@orangejenny

**Comment by orangejenny** [comment]
@mkangia Thank you for testing and updating. I will coordinate with you when this gets released (guessing that won't be until next week, because we're going to do some more testing).

**Pull request merged**